tasks/multiple-regression/src/main.rs

```diff
@@ -0,0 +1,50 @@
+use nalgebra::DMatrix;
+/// For a given a set of data vectors in the following format y = {y₁, y₂, ... yn}
+/// and X = {xi₁, xi₂, ... xin}, where i 𝜖 1..k
+///
+/// computes the vector β = {β₁, β₂, ...βk} using ordinary least squares regression
+fn get_coefficients(x: &DMatrix<f64>, y: &DMatrix<f64>) -> DMatrix<f64> {
+    (x.transpose() * x).try_inverse().unwrap() * x.transpose() * y
+}
+
+#[allow(clippy::cast_possible_truncation)]
+#[allow(clippy::cast_possible_wrap)]
+fn main() {
+    let x = vec![
+        1.47_f64, 1.50, 1.52, 1.55, 1.57, 1.60, 1.63, 1.65, 1.68, 1.70, 1.73, 1.75, 1.78, 1.80,
+        1.83,
+    ];
+
+    let y = vec![
+        52.21_f64, 53.12, 54.48, 55.84, 57.20, 58.57, 59.93, 61.29, 63.11, 64.47, 66.28, 68.10,
+        69.92, 72.19, 74.46,
+    ];
+
+    // generate 1, x, and x^2
+    let x = DMatrix::from_fn(x.len(), 3, |row, col| x[row].powi(col as i32));
+    let y = DMatrix::from_vec(y.len(), 1, y);
+    println!(
+        "\u{3B2} parameters using least squares regression :{}",
```

Comment:
```suggestion
        "β parameters using least squares regression :{}",
```

Reply:
Well, it was like that, but then pedantic clippy said [this](https://rust-lang.github.io/rust-clippy/master/index.html#non_ascii_literal)

Reply:
Hm, I disagree with that lint. I don't think we should be running pedantic clippy lints.

Reply:
Ok, no worries, I'll update this.